woocommerce/woocommerce-gutenberg-products-block
# Product Gallery Block: Add zoom on hovering on the frontend

~~⚠️ This PR is blocked by #10764.~~

## What

Fixes #9941

## Why



This PR adds the zoom when the image is hovered, thanks to the Interactivity API. Please review #10795.

## Testing Instructions



_Please consider any edge cases this change may have, and also other areas of the product this may impact._

1. Open the Site Editor. Visit the Single Product Template.
2. Add the Product Gallery Block.
3. Ensure that the setting about the zoom on hovering is enabled.
4. Save.
5. Visit a product page.
6. Hover the image and ensure that the image is zoomed on hovering.
7. Visit the Single Product Template.
8. Disable the setting about the zoom on hovering.
9. Visit a product page.
10. Ensure that the image isn't zoomed on hover.


## Screenshots or screencast




https://github.com/woocommerce/woocommerce-blocks/assets/4463174/2faa862e-af81-4730-85eb-0e411e7ea488



## WooCommerce Visibility


Required:


## Checklist

Required:

Conditional:

## Changelog


> Add suggested changelog entry here.
